apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/index.php

```diff
@@ -76,27 +76,14 @@ function enqueue_launch_button_script_and_style( $site_launch_options ) {
  * @param array $site_launch_options Site launch options.
  */
 function enqueue_launch_flow_script_and_style( $site_launch_options ) {
-
-	$launch_flow = $site_launch_options['launch_flow'];
-
-	// Determine script name by launch flow.
-	// We are avoiding string concatenation for security reasons.
-	switch ( $launch_flow ) {
-		case 'gutenboarding-launch':
-			$script_name = 'gutenboarding-launch';
-			break;
-		case 'focused-launch':
-			$script_name = 'focused-launch';
-			break;
-		default:
-			// For redirect or invalid flows, skip & exit early.
-			return;
-	}
```

Comment:
This feels a bit like scope creep since it isn't necessary for removing the launch button.